e2e/cmd/run_tests/03_test_non_pro_managers/main.go

```diff
@@ -0,0 +1,116 @@
+// Package main provides a script that applies and asserts non-Pro policies on
+// the provisioned Ubuntu client.
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ubuntu/adsys/e2e/internal/command"
+	"github.com/ubuntu/adsys/e2e/internal/inventory"
+	"github.com/ubuntu/adsys/e2e/internal/remote"
+)
+
+var sshKey string
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	cmd := command.New(action,
+		command.WithValidateFunc(validate),
+		command.WithRequiredState(inventory.ADProvisioned),
+	)
+	cmd.Usage = fmt.Sprintf(`go run ./%s [options]
+
+Apply and assert non-Pro policies on the Ubuntu client.
+
+These policies are configured in the e2e/assets/gpo directory, and described as
+part of the ADSys QA Plan document.
+
+This script will:
+ - reboot the client VM to trigger machine policy application
+ - assert machine GPO rules were applied
+ - assert users and admins GPO rules were applied
+
+The run is considered successful if the script exits with a zero exit code.
+
+The runner must be connected to the ADSys E2E tests VPN.`, filepath.Base(os.Args[0]))
+
+	return cmd.Execute(context.Background())
+}
+
+func validate(_ context.Context, cmd *command.Command) error {
+	var err error
+	sshKey, err = command.ValidateAndExpandPath(cmd.Inventory.SSHKeyPath, command.DefaultSSHKeyPath)
```

Comment:
We could use named return instead and avoid having the `var err error` line, but I'm not 100% sure what's the best approach here.